### PR TITLE
Search

### PR DESCRIPTION
**Pull request by dboston528**

### CODE REVIEW

File: src/common/Navbar.js [hunk 1]
```diff
@@ -264,11 +265,13 @@ class Navbar extends Component {
                   </div>
                 </div>
                 <div className="input-group">
+                 {/* <SearchBar
+                />  */}
```
Comment (vhoof) [inline src/common/Navbar.js]: Why are we commenting this out here and continuing to use the `AutoComplete` component below?

File: src/common/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+} 
+    
+  
+
+  document.addEventListener('DOMContentLoaded', () => {
+    ReactDOM.render(
+      <SearchBar />,
+      document.body.appendChild(document.createElement('div')),
+    )
+  });
```
Comment (vhoof) [inline src/common/SearchBar.js]: I'm not sure why we need to do this vs. just embedding the component.

**Comment by vhoof** [comment]
You'll want to resolve merge conflicts, too.

File: src/common/Navbar.js (hunk 1 above)
Comment (dboston528) [reply src/common/Navbar.js]: I changed it back.

File: src/common/SearchBar.js (hunk 2 above)
Comment (dboston528) [reply src/common/SearchBar.js]: I deleted it

**Pull request merged**